src/oidcop/oidc/token.py

```diff
@@ -281,6 +278,19 @@ def process_request(self, req: Union[Message, dict], **kwargs):
 
         token.register_usage()
 
+        if ("client_id" in req
+            and req["client_id"] in _context.cdb
+            and "revoke_refresh_on_issue" in _context.cdb[req["client_id"]]
+        ):
+            revoke_refresh = _context.cdb[req["client_id"]].get("revoke_refresh_on_issue")
+        else:
+            revoke_refresh =(
+                _mngr.conf["token_handler_args"]["refresh"]["kwargs"].get("revoke_refresh_on_issue", False)
```

Comment:
Why add it to the token handler? Do you think it's a better there than the token class? You could add it to the token endpoint (oidcop/{oauth2,oidc}/token.py:Token) and then we could have here:
```python
revoke_refresh = self.endpoint.revoke_refresh_on_issue
```
what do think about this?

Reply:
I thought token handler as a more suitable place as it describes the behavior upon tokens, in contrast with token class that describe the actual token.

The truth is that in oauth2/token.py: https://github.com/IdentityPython/oidc-op/blob/66eb57b305258308365b5439bb1f41c32950018b/src/oidcop/oauth2/token.py#L346 there is an unused parameter `new_refresh_token` that it seems to resembles the same functionality as `revoke_refresh`.

So I have no reason to insist on this decision.